docs/source/demonstrations/materials/tetrahedra_mesh_emitter.rst

```diff
@@ -0,0 +1,12 @@
+.. _demo_custom_volume_emitters:
```

Comment:
Just noticed a small issue here. This reference label is copy-pasted from volume_inhomogeneous.rst. These labels must be unique in each .rst file, or cross-references will not work. 
Also, you have to modify docs/source/demonstrations.rst if you want this demo to be listed.

Reply:
I changed the reference label and modified demonstrations.rst a little.